client/src/Pos/Client/Txp/Balances.hs

```diff
@@ -53,6 +55,9 @@ getOwnUtxo genesisData = getOwnUtxos genesisData . one
 getOwnUtxoForPk :: MonadBalances m => GenesisData -> PublicKey -> m Utxo
 getOwnUtxoForPk genesisData ourPk = getOwnUtxos genesisData ourAddresses
   where
+    getNM :: GenesisData -> NetworkMagic
+    getNM = makeNetworkMagic . gpcProtocolMagic . gdProtocolConsts
```

Comment:
This could also be done
```
  where
    nm :: NetworkMagic
    nm = makeNetworkMagic . gpcProtocolMagic . gdProtocolConsts $ genesisData
...
    ourAddresses =
        map (flip (makePubKeyAddress nm) ourPk . IsBootstrapEraAddr) [False, True]
```

Since only use `getNM` once, we don't necessarily need to abstract out a projection function.

Reply:
This isn't a big deal - I'm more making a note for future-development-practices.